core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -264,9 +264,8 @@ object Boot extends IOApp with LazyLogging {
         gcsConfig.getString("notifications.topicName")
       )
 
-      val marthaConfig: Config = conf.getConfig("martha")
-      val marthaBaseUrl: Option[String] = marthaConfig.getStringOption("baseUrl")
-      val marthaUrl = marthaBaseUrl.map(_ + "/martha_v3").getOrElse(marthaConfig.getString("baseUrl_v2"))
+      val marthaBaseUrl: String = conf.getConfig("martha").getString("baseUrl")
+      val marthaUrl: String = marthaBaseUrl.flatMap(_ + "/martha_v3")
```

Comment:
flatMapping on a String would split the string into characters and append `/martha_v3` at the end of each character. 
Example:
```
val strVal = "hello"
val path = strVal.flatMap(_ + "/world")
println(path)
```
Result:
```
h/worlde/worldl/worldl/worldo/world
```

Instead one can do 
```
val marthaUrl: String = s"$marthaBaseUrl/martha_v3"
```

Reply:
Thank you!  Done.